backend/src/Manager/Subscription/SubscriptionDetailsManager.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Manager\Subscription;
+
+use App\AutoMapping;
+use App\Entity\SubscriptionDetailsEntity;
+use App\Repository\SubscriptionDetailsEntityRepository;
+use App\Request\Subscription\SubscriptionDetailsCreateRequest;
+use App\Request\Subscription\SubscriptionUpdateRequest;
+use App\Request\Subscription\SubscriptionRemainingOrdersUpdateRequest;
+use App\Request\Subscription\SubscriptionRemainingTimeUpdateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+
+class SubscriptionDetailsManager
+{
+    public function __construct(private AutoMapping $autoMapping, private EntityManagerInterface $entityManager, private SubscriptionDetailsEntityRepository $subscribeDetailsRepository)
+    {
+        $this->autoMapping = $autoMapping;
+        $this->entityManager = $entityManager;
+        $this->subscribeDetailsRepository = $subscribeDetailsRepository;
+    }
+
+    public function createSubscriptionDetails($subscription):SubscriptionDetailsEntity
```

Comment:
I think we need to define the type of the var _subscription_,

Reply:
done